docs/cli/installation.mdx

```diff
@@ -7,6 +7,14 @@ pagination_next: null
 pagination_prev: null
 ---
 
+import useDocusaurusContext from '@docusaurus/useDocusaurusContext';
+export function Version() {
+  const {
+    siteConfig: {customFields},
+  } = useDocusaurusContext();
+  return <span>"{customFields.version}"</span>;
```

Comment:
@ca7alindev Please remove <span> it will breaks CLI command

Reply:
@evghen1 remove entire component from mdx and create it in new file?

Reply:
@ca7alindev possible to be done like this:
```
import {customFields} from '@site/src/constants';

This is the value: {customFields.foo}.
```

Reply:
@evghen1 i found this, but require a new file in src with constants

Reply:
@ca7alindev sure, but better component like i described in other comment: 
https://github.com/kndpio/website/pull/17#discussion_r1576181587